renovate-config-iac.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "description": "Shared Renovate configuration for dsb-norge IaC projects where major updates are skipped",
+    "enabledManagers": [
+        "terraform",
+        "tflint-plugin"
+    ],
+    "extends": [
+        "helpers:pinGitHubActionDigests",
+        ":separateMultipleMajorReleases",
+        ":semanticCommits",
+        ":semanticPrefixFixDepsChoreOthers",
+        ":prConcurrentLimit10"
+    ],
+    "minimumReleaseAge": "3 days",
+    "prCreation": "not-pending",
+    "terraform": {
+        "ignorePaths": [
+            "**/_terraform-state/**"
+        ],
+        "pinDigests": true
+    },
+    "commitMessageLowerCase": "never",
+    "prHourlyLimit": 4,
+    "major": {
+        "enabled": true
+    },
+    "ignoreScripts": true,
```

Comment:
This is applicable only for npm, bun, Composer and Copier. ref. https://docs.renovatebot.com/configuration-options/#ignorescripts